### PR TITLE
Add option to omit levels from all items or only non heading items

### PR DESCRIPTION
**Pull request by michaelnt**
PR to close issue #211.

I left in the -L option but removed the --no-body-levels option so shouldn't break existing use too much.

### CODE REVIEW

**Comment by jacebrowning** [comment]
Looks pretty good so far. Wamt to add a test here? https://github.com/jacebrowning/doorstop/blob/46b108d912673a896e5d830d78b6681422137b77/doorstop/core/test/test_publisher.py#L14

File: doorstop/cli/test/test_all.py [hunk 1]
```diff
@@ -699,9 +699,15 @@ def test_publish_document_without_child_links(self):
 
     def test_publish_document_without_body_levels(self):
         """Verify 'doorstop publish' can create output without body levels."""
-        self.assertIs(None, main(['publish', 'tut', '--no-body-levels']))
```
Comment (jacebrowning) [inline doorstop/cli/test/test_all.py]: To ensure that no one's existing workflow breaks, `--no-body-levels` should still be support and this should become two tests, one for `--no-levels=body` and another for the legacy method.

File: doorstop/cli/utilities.py [hunk 2]
```diff
@@ -112,8 +112,11 @@ def configure_settings(args):
     # Parse `publish` settings
     if hasattr(args, 'no_child_links') and args.no_child_links is not None:
         settings.PUBLISH_CHILD_LINKS = args.no_child_links is False
-    if hasattr(args, 'no_body_levels') and args.no_body_levels is not None:
-        settings.PUBLISH_BODY_LEVELS = args.no_body_levels is False
+    if hasattr(args, 'no_body_levels'):
+        settings.PUBLISH_BODY_LEVELS = not args.no_body_levels
```
Comment (jacebrowning) [inline doorstop/cli/utilities.py]: Please add a deprecation warning here so this option can be safely removed in the 2.0 release:

```python
msg = "'--no-body-levels' option will be removed in a future release"
warnings.warn(msg, DeprecationWarning)
```

Comment (michaelnt) [inline doorstop/cli/utilities.py]: I've made that change, the warning doesn't get printed by default. 

I could see the warning and some other warnings if I run python -Wd

**Pull request merged**